analytics/population/README.md

```diff
@@ -0,0 +1,30 @@
+<div style = "text-align: justify"> 
+  
+# **Updated population density of the LA city neighborhood councils (NCs).**
+  
+### Have calculated the population of LA city neighborhood councils using the updated NC shape file, Census tracts data-2020 and 2020-ACS (5 year estimate) demographics data. The notebook- NC_pop_recent.ipynb outlines the detailed steps involved to evaluate the population, area (in square miles) and population density of the 99 NCs. 
```

Comment:
We don't need to use `###` here for any of the paragraphs.

Reply:
Thanks Nich! Removed them.